packages/uicore/src/components/Carousel/Carousel.tsx

```diff
@@ -28,10 +30,19 @@ export const Carousel: React.FC<CarouselProps> = ({
   nextElement,
   previousElement,
   hideIndicators = false,
+  autoPlay,
+  autoPlayDuration = 2000,
```

Comment:
Nit: Something like `autoPlayInterval` or `autoPlayTimeout` gives a better indication of what this prop does.
`autoPlayDuration` implies  duration for which autplay is active?

Also could you add annotation to these props, it helps with storybook and autocomplete in VS Code.